virl2_client/models/node.py

```diff
@@ -136,7 +138,7 @@ def __str__(self):
     def __repr__(self):
         return (
             "{}({!r}, {!r}, {!r}, {!r}, {!r}, {!r}, {!r}, "
-            "{!r}, {!r}, {!r}, {!r}, {!r}, {!r}, {!r}, {!r})".format(
+            "{!r}, {!r}, {!r}, {!r}, {!r}, {!r}, {!r}, {!r}, {!r})".format(
```

Comment:
this needs to be changed, but there's no need to add parameters, please remove them